WcaOnRails/app/models/registration.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 class Registration < ActiveRecord::Base
-  scope :pending, -> { where(accepted_at: nil) }
-  scope :accepted, -> { where.not(accepted_at: nil) }
+  scope :pending, -> { where(accepted_at: nil).where(deleted_at: nil) }
+  scope :accepted, -> { where.not(accepted_at: nil).where(deleted_at: nil) }
+  scope :deleted, -> { where.not(deleted_at: nil) }
```

Comment:
I'm not sure we need a default scope at the moment because the code always seems to  look for accepted or pending.